.github/workflows/node.js.yml

```diff
@@ -90,18 +90,20 @@ jobs:
       - name: Build
         run: npm run build
 
+      # Run unit tests on all files for accurate code coverage
+      # Lint only changed files so linting is faster
       - name: Test (Pull Request)
         if: ${{ github.event_name == 'pull_request' }}
         run: |
-          npm run test:unit -- --cacheDirectory node_modules/.cache/jest
-          npm run test:lint -- --cacheDirectory node_modules/.cache/jest --changedSince origin/${{ github.event.pull_request.base.ref }}
+          npm run test:unit -- --maxWorkers=50% --cacheDirectory node_modules/.cache/jest
```

Comment:
This option doesn't seem to have much of an impact, so I suggest we omit the option so we don't have to explain it later